resources/apt-packages.txt

```diff
@@ -30,3 +30,11 @@ libblas-dev
 libfreetype6-dev  
 liblapack-dev
 libquadmath0
+# Needed by pylibmc
```

Comment:
"Really high demand", "required for the product to work", or "we have a specific use case in the documentation that needs this". In this case, its all three. We need this for the Memcache documentation and sample, which is a critical feature.